<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>505</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="testFileButton">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Select Test List File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="testFileLabel">
       <property name="text">
        <string>Selected Test List File:  None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="metaFileButton">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Select Meta File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="metaFileLabel">
       <property name="text">
        <string>Selected Meta File:  None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="modelFileButton">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Load Pretrained Model</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="modelFileLabel">
       <property name="text">
        <string>Selected Pretrained Model:  None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Classification</string>
    </property>
   </widget>
   <widget class="QLabel" name="classificationResultLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Classification result:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="setDataPathButton">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Set Data Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataPathLabel">
       <property name="text">
        <string>Selected Data Path: None</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="initModelButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Init Model</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>210</y>
      <width>331</width>
      <height>35</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="setDataIdButton">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>35</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Set Data Id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Select No. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="chooseNumberEdit">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="numTotalLabel">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>/0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
